includes/create-theme/theme-patterns.php

```diff
@@ -4,23 +4,44 @@ class CBT_Theme_Patterns {
 	public static function pattern_from_template( $template, $new_slug = null ) {
 		$theme_slug      = $new_slug ? $new_slug : wp_get_theme()->get( 'TextDomain' );
 		$pattern_slug    = $theme_slug . '/' . $template->slug;
-		$pattern_content = (
-		'<?php
-/**
- * Title: ' . $template->slug . '
- * Slug: ' . $pattern_slug . '
- * Categories: hidden
- * Inserter: no
- */
-?>
-' . $template->content
-		);
+		$pattern_content = <<<PHP
+		<?php
+		/**
+		 * Title: {$pattern->title}
+		 * Slug: {$pattern->slug}
+		 * Categories: {$pattern->categories}
+		 */
+		?>
+		{$pattern_post->post_content}
```

Comment:
```suggestion
		 * Title: {$template->title}
		 * Slug: {$pattern_slug}
		 * Categories: {$template->categories}
		 */
		?>
		{$template->post_content}
```

I think this should use `$template` rather than `$pattern_post` - I'm guessing this was accidentally copied over from `pattern_from_wp_block`. `$template->categories` might need to be created, too.